app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -56,7 +58,6 @@ const AppLayoutWithExperiment = ({
   // is complete and we inspect its status. To prevent content from flashing on the screen
   // between the experiment and analysis requests, assume we need to wait for analysis
   // until we can prove otherwise.
```

Comment:
This comment should be able to be removed (or reworded) since it's not longer relevant.